baldrick/plugins/github_towncrier_changelog.py

```diff
@@ -146,31 +149,36 @@ def process_towncrier_changelog(pr_handler, repo_handler, headers):
 
     modified_files = pr_handler.get_modified_filenames()
 
-    messages = []
+    messages = {}
+
     matching_file = check_sections(modified_files, section_dirs)
-    if not matching_file:
-        messages.append(cl_config.get("missing_file_message", NO_CHANGELOG))
+
+    if skip_label and skip_label in pr_handler.labels:
+
+        pass
+
+    elif not matching_file:
+
+        messages['missing_file'] = {'message': NO_CHANGELOG, 'status': 'failure'}
```

Comment:
`NO_CHANGELOG` should be a fall back to the config value `cl_config.get("missing_file_message", NO_CHANGELOG)`

Reply:
Done (and the others below)